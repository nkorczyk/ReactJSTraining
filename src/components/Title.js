import React from 'react';

const Title = () => {
  return (
    <div>
      <h5 className="red-text text-darken-2">netflixroulette</h5>
    </div>
  )
}

export default Title;
