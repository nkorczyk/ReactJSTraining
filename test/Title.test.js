import React from 'react';
import { mount } from 'enzyme';
import Title from '../src/components/Title';

describe('should render Title component', () => {
  const element = mount(<Title />);
  it('Snapshot test with default props', () => {
    expect(element).toMatchSnapshot();
  });
});

it('should render netflixroulette string', () => {
  const element = mount(<Title />);
  expect(element.text()).toEqual('netflixroulette');
});
