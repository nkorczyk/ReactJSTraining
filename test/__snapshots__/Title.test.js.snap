// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Title component Snapshot test with default props 1`] = `
<Title>
  <div>
    <h5
      className="red-text text-darken-2"
    >
      netflixroulette
    </h5>
  </div>
</Title>
`;
